yarn-project/noir-contracts/src/contracts/child_contract/src/main.nr

```diff
@@ -3,24 +3,20 @@ contract Child {
     use dep::aztec3::abi::Inputs;
     use dep::aztec3::context::PrivateFunctionContext;
 
-    /// ABI constructor type "secret"
-    /// ABI constructor params []
-    /// ABI constructor return []
     fn constructor(
         inputs: pub Inputs,
     ) -> pub [Field; dep::aztec3::abi::PUBLIC_INPUTS_LENGTH] {
         PrivateFunctionContext::new().finish(inputs)
     }
 
-    /// ABI value type "secret"
-    /// ABI value params [{"name":"input","type":{"kind":"field"},"visibility":"public"}]
-    /// ABI value return [{"kind":"field"}]
```

Comment:
Shouldn't we keep the `return` annotation at least?

Reply:
Users currently cannot consume the return value for secret functions! The idea is to phase out the ABI manual comments for the introduction of the aztec noir compiler :smile: